packages/editor/src/components/widgets/table/footer/TableFooter.tsx

```diff
@@ -18,3 +18,16 @@ export const TableAddRow = (props: { colSpan: number; addRow: () => void }) => {
     </tr>
   );
 };
+
+export const TableShowMore = (props: { colSpan: number; showMore: () => void; helpertext: string }) => {
+  const readonly = useReadonly();
+  return (
+    <tr>
+      <th colSpan={props.colSpan} className='show-more-rows'>
+        <Button rotate={90} onClick={props.showMore} disabled={readonly} aria-label='Show more rows'>
+          ...{props.helpertext}
```

Comment:
why do you not use the `dots` ivy icon?

Reply:
Yes, I could do that, but I need to include more CSS magic for the icon to look the same. Otherwise, it would look something like this:
![grafik](https://github.com/axonivy/inscription-client/assets/141223521/f46e0b6c-bccc-415d-b110-33eaab919f1c)
or this:
![grafik](https://github.com/axonivy/inscription-client/assets/141223521/49820e0a-3cb0-4e72-b380-28f831ddc0c4)

and without the icon it just looks like this:
![grafik](https://github.com/axonivy/inscription-client/assets/141223521/525669e3-0c57-4f74-b4ea-1ca1d859132e)

what do you prefer?

Reply:
I had the second in mind. Not sure what you mean with a lot more css magic, but if it isn't worth (in your opinion), I would just show the text without the dots or any icon.